HW3/Anastasiia-web/2.py

```diff
@@ -0,0 +1,67 @@
+# TASK 2

```

Comment:
number = input("Enter the number:")
print("Reversed number is:", number[::-1])
print("Sorted number is:", sorted(number))

prod=1
while number>0:
    digit = number%10
    prod *= digit
    number //= 10
print("The product of all numbers", prod)